src/test/java/com/officemanagement/resource/BaseResourceTest.java

```diff
@@ -113,6 +113,7 @@ private void cleanDatabase() {
         session.createNativeQuery("SET REFERENTIAL_INTEGRITY FALSE").executeUpdate();
         
         // Truncate all tables
+        session.createNativeQuery("DELETE FROM employee_seats").executeUpdate();
```

Comment:
this is the fix I added. I need to understand why in local the tests were working